specification/msi/resource-manager/readme.md

```diff
@@ -1,30 +1,43 @@
 # Managed Service Identity
+
 > see https://aka.ms/autorest
 
 This is the AutoRest configuration file for Managed Service Identity.
 
 ---
+
 ## Getting Started
+
 To build the SDK for Managed Service Identity, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
 
 > `autorest`
 
 To see additional help and options, run:
 
 > `autorest --help`
+
 ---
 
 ## Configuration
 
-
 ### Basic Information
+
 These are the global settings for the Managed Service Identity API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2018-11-30
+tag: package-preview-2021-09
 ```
 
+
+### Tag: package-preview-2021-09
```

Comment:
> Public private branch 'msi-associated-resources-preview-api-2021-09-30'

I guess the inconsistency between the api-version and folder name is caused by the tag name of the SDK should be added with the date information, otherwise the folder name of the SDK version will default to `2021_09_01`